sql/src/main/java/io/crate/planner/node/ddl/AlterBlobTablePlan.java

```diff
@@ -0,0 +1,89 @@
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.planner.node.ddl;
+
+import io.crate.analyze.AlterTableAnalyzedStatement;
+import io.crate.analyze.AnalyzedAlterBlobTable;
+import io.crate.analyze.SymbolEvaluator;
+import io.crate.analyze.TableParameter;
+import io.crate.analyze.TableParameters;
+import io.crate.data.Row;
+import io.crate.data.Row1;
+import io.crate.data.RowConsumer;
+import io.crate.execution.support.OneRowActionListener;
+import io.crate.expression.symbol.Symbol;
+import io.crate.metadata.table.TableInfo;
+import io.crate.planner.DependencyCarrier;
+import io.crate.planner.Plan;
+import io.crate.planner.PlannerContext;
+import io.crate.planner.operators.SubQueryResults;
+import io.crate.sql.tree.AlterTable;
+
+import java.util.function.Function;
+
+import static io.crate.planner.node.ddl.AlterTablePlan.getTableParameter;
+import static io.crate.planner.node.ddl.AlterTablePlan.maybeRaiseBlockedException;
+
+public class AlterBlobTablePlan implements Plan {
+
+    private final AnalyzedAlterBlobTable analyzedAlterTable;
+
+    public AlterBlobTablePlan(AnalyzedAlterBlobTable analyzedAlterTable) {
+        this.analyzedAlterTable = analyzedAlterTable;
+    }
+
+    @Override
+    public StatementType type() {
+        return StatementType.DDL;
+    }
+
+    @Override
+    public void executeOrFail(DependencyCarrier dependencies,
+                              PlannerContext plannerContext,
+                              RowConsumer consumer,
+                              Row params, SubQueryResults subQueryResults) {
+        Function<? super Symbol, Object> eval = x -> SymbolEvaluator.evaluate(
+            plannerContext.transactionContext(),
+            plannerContext.functions(),
+            x,
+            params,
+            subQueryResults
+        );
+
+        TableInfo tableInfo = analyzedAlterTable.tableInfo();
+        AlterTable<Object> alterTable = analyzedAlterTable.alterTable().map(eval);
+
+        TableParameter tableParameter = getTableParameter(alterTable, TableParameters.ALTER_BLOB_TABLE_PARAMETERS);
+        maybeRaiseBlockedException(tableInfo, tableParameter.settings());
+
+        AlterTableAnalyzedStatement stmt = new AlterTableAnalyzedStatement(
```

Comment:
would it make sense to get rid of the `AlterTableAnalyzedTable` and adapt `executeAlterTable` to work with `AlterTable<Object>` + `TableParameter` ?

Reply:
The `AlterTableAnalyzedStatement` is used inside the analyzer tests. We could maybe refactor them to validate on a request object, I'd suggest to do this in a follow up.

Reply:
Maybe we could rename it meanwhile to `BoundAlterTableStatement` or so, to avoid confusing between the two alter table statement classes.